microprofile-telemetry-metrics/src/main/java/org/jboss/eap/qe/microprofile/metrics/namefellow/PingOneService.java

```diff
@@ -1,17 +1,34 @@
 package org.jboss.eap.qe.microprofile.metrics.namefellow;
```

Comment:
As said, I'd refactor the package from `org.jboss.eap.qe.microprofile.metrics.namefellow` to `org.jboss.eap.qe.microprofile.telemetry.metrics.namefellow`

Reply:
+1